### ISSUE TITLE
pypanddoc and pySpark not installing from pip

### ISSUE THREAD

**Issue by jpdna** [bigdatagenomics/mango#434]
When I most recently tried to run `make prepare` in mango-python I now get
```
    Complete output from command python setup.py egg_info:
    Could not import pypandoc - required to package PySpark
    Download error on https://pypi.org/simple/pypandoc/: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590) -- Some packages may not be found!
    Couldn't find index page for 'pypandoc' (maybe misspelled?)
    Download error on https://pypi.org/simple/: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590) -- Some packages may not be found!

```
on the master branch, where previously this worked fine.
I think this requires an update to my local pip, but so far I've tried that and still get the same error which running `make prepare`.   
I assume this a change that happened at the pip repository that creates this problem for clients which are now incompatible, though pip does seems to work for many of the other required packages, so if may be specific to pypandoc.

If someone else is able to clone the repo at Master and get `mango-python/make prepare` to run then please let me know your  version of pip and python.

**Comment by jpdna** [comment]
I did get this to work again after installing Python 2.7.15 and compiling with openSSL.  That version came with pip version 18.0.  I am running a virtualenv.

The previously version I was using was Python 2.7.10 on MacOSX with pip version 18.0  in a vitualenv, which seemed to be working fine as of earlier this week, but starting today gave me the error above.

Closing this ticket given that it works for me now that I upgraded to most current Python 2, but would not be surprised if another user with an older version of python or pip might experience the same problem.
